packaging/suse/amazon-ecs-init.spec

```diff
@@ -0,0 +1,78 @@
+#
+# spec file for package amazon-ecs-init
+#
+# Copyright (c) 2015 SUSE LINUX Products GmbH, Nuernberg, Germany.
+#
+# All modifications and additions to the file contributed by third parties
+# remain the property of their copyright owners, unless otherwise agreed
+# upon. The license for this file, and modifications and additions to the
+# file, is the same license as for the pristine package itself (unless the
+# license for the pristine package is not an Open Source License, in which
+# case the license is the MIT License). An "Open Source License" is a
+# license that conforms to the Open Source Definition (Version 1.9)
+# published by the Open Source Initiative.
+
+# Please submit bugfixes or comments via http://bugs.opensuse.org/
+#
+
+Name:           amazon-ecs-init
+Version:        1.3.1
```

Comment:
Are you packaging 1.3.1?  The current version we have is 1.4.0 and I'd like to make sure that whatever version indicated in the spec file matches the version of the Agent that gets loaded into Docker.